conf/nginx/nginx.conf

```diff
@@ -40,6 +40,13 @@ http {
             proxy_intercept_errors on;
             error_page 301 302 307 = @handle_redirect;
 
+            # Handle most errors upstream listed at:
+            # https://www.restapitutorial.com/httpstatuscodes.html
+            error_page 404 410 = @proxy_not_found;
+            error_page 420 429 = @proxy_too_many_requests;
+            error_page 400 401 402 403 405 406 407 408 409 411 412 413 414 415 416 417 418 422 423 424 425 426 428 431 444 449 450 451 = @proxy_client_error;
+            error_page 500 501 502 503 504 505 506 507 508 509 510 511 598 599 = @proxy_upstream_error;
```

Comment:
Some attempt has been made to split out different cases we might find interesting. In particular "too many requests" type errors could be very interesting to us, and would be otherwise mixed in with other 400s. 404's also seem like a useful thing to reply to the user.

As far as I can tell, there's no way of using pattern matching here to say `5**`. So we have to list everything we can think of.